.gitignore

```diff
@@ -12,7 +12,7 @@ homedocs
 jsdoc-template
 api
 package-lock.json
-lib
+#lib
```

Comment:
We don't publish this package so I just added the build folder into the repo so it gets copied when pulled down.

Reply:
That is: ignore. the entire 'lib' folder please